packages/perseus/src/widgets/interactive-graphs/graphs/components/angle.tsx

```diff
@@ -123,6 +134,20 @@ export const Angle = ({
     );
 };
 
+const RightAngleArc = ({start: [x1, y1], vertex: [x2, y2], end: [x3, y3]}) => (
+    <g
+        style={{
+            transform: `var(--mafs-view-transform) var(--mafs-user-transform)`,
+        }}
+    >
```

Comment:
Suggestion: the group with the transform is duplicated here and on line 105 above. Maybe the `g` element could be moved outside the ternary that chooses the square or arc—something like:

```ts
<g
  style={{
    transform: `var(--mafs-view-transform) var(--mafs-user-transform)`,
  }}
>
  {!isInside && isRightAngle(angle) ? (
    <RightAngleArc
      // ...
    />
  ) : (
    <path d={arc} strokeWidth={0.02} fill="none" />
  )}
</g>
```

Actually, since only the `d` property of the path is different, maybe it could even be:

```ts
<g
  style={{
    transform: `var(--mafs-view-transform) var(--mafs-user-transform)`,
  }}
>
  <path
    strokeWidth={0.02}
    fill="none"
    d={
      !isInside && isRightAngle(angle)
        ? square
        : arc
    }
  />
</g>
```

Reply:
I'm using the right angle square elsewhere-- see the early return in the code above